html-report/src/components/HomePage.js

```diff
@@ -34,7 +34,7 @@ export default class HomePage extends Component {
                                     </div>
                                     <div className="card-info">
                                         <div className="text-sub-title-light">Duration</div>
-                                        <div className="card-info__content">{convertTime(pool.duration_millis)}</div>
```

Comment:
You're essentially adding a link to the timeline for all executed tests, but the duration is shown only for one pool of devices. Also discovering this would be problematic. Would it be ok if we just add the link to the top of the page near POOLS? I just don't think the context is correct in this case.

Reply:
Sure I have no strong opinion on the location. I can change this around